File: 백준/Bronze/9772. Quadrants/Quadrants.cc
#include <iostream>
using namespace std;
int main(void) {
    double x, y;
    while (cin >> x >> y) {
        if (x == 0 || y == 0)
            cout << "AXIS\n";
        else
            cout << (x > 0 ? (y > 0 ? "Q1" : "Q4") : (y > 0 ? "Q2" : "Q3")) << "\n";
    }
    return 0;
}

File: 백준/Bronze/9772. Quadrants/Quadrants.py
import sys
input=lambda:sys.stdin.readline().rstrip()
while True:
    t=input()
    if t=="": break
    x,y=map(float,t.split())
    if x==0 or y==0:
        print("AXIS")
    else:
        print(("Q1" if y>0 else "Q4") if x>0 else ("Q2" if y>0 else "Q3"))